src/pages/Follow/index.tsx

```diff
@@ -30,6 +30,7 @@ const filterSendOrReceiveInfo = (
     nickname: isReceive ? item.user.nickname : item.follower.nickname,
     userType: isReceive ? item.user.userType : item.follower.userType,
     requestId: item.id,
+    profileImage: isReceive ? item.user.profileImage : item.follower.profileImage,
   }));
```

Comment:
조건문을 따로 빼서 코드를 줄이는 게 더 가독서이 좋을 것 같아요!

```suggestion
  const filteredData: FollowerInfo[] = data.content.map((item) => {
  const selectedUser = isReceive ? item.user : item.follower;

  return {
    account: selectedUser.account,
    id: selectedUser.id,
    key: selectedUser.id,
    followedType: isReceive ? 'REQUEST_RECEIVE' : 'REQUEST_SENT',
    nickname: selectedUser.nickname,
    userType: selectedUser.userType,
    requestId: item.id,
    profileImage: selectedUser.profileImage,
  };
});

```

Reply:
시간 관계상 위 이슈들은 배포 후 수정하도록 하겠습니다!!
감사합니다